WordPress/Classes/ViewRelated/NUX/SignupUsernameTableViewController.swift

```diff
@@ -0,0 +1,237 @@
+import SVProgressHUD
+
+protocol SignupUsernameTableViewControllerDelegate {
+    func usernameSelected(_ username: String)
+    func newSearchStarted()
+}
+
+class SignupUsernameTableViewController: NUXTableViewController {
+    open var currentUsername: String?
+    open var displayName: String?
+    open var delegate: SignupUsernameTableViewControllerDelegate?
+    private var service: AccountSettingsService?
+    private var suggestions: [String] = []
+    private var isSearching: Bool = false
+    private var selectedCell: UITableViewCell?
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        tableView.register(UINib(nibName: "SiteCreationDomainSearchTableViewCell", bundle: nil), forCellReuseIdentifier: SiteCreationDomainSearchTableViewCell.cellIdentifier)
+        setupBackgroundTapGestureRecognizer()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        WPStyleGuide.configureColors(for: view, andTableView: tableView)
+        tableView.layoutMargins = WPStyleGuide.edgeInsetForLoginTextFields()
+
+        navigationItem.title = NSLocalizedString("Pick username", comment: "Title for selecting a new username in the site creation flow.")
```

Comment:
The navigation bar doesn't actually show up:
<img width="415" alt="screen shot 2018-02-25 at 11 08 37 am" src="https://user-images.githubusercontent.com/1816888/36644762-47ff348c-1a1c-11e8-9e5d-2366f2b52662.png">

OMG I can't go back!!! 😉